README.md

```diff
@@ -60,6 +60,7 @@ lib('java.RemoveUnusedImportsStep')              +'{{yes}}       | {{yes}}
 extra('java.EclipseJdtFormatterStep')            +'{{yes}}       | {{yes}}      | {{yes}}      | {{no}}  |',
 lib('kotlin.KtLintStep')                         +'{{yes}}       | {{yes}}      | {{yes}}      | {{no}}  |',
 lib('kotlin.KtfmtStep')                          +'{{yes}}       | {{yes}}      | {{no}}       | {{no}}  |',
+lib('kotlin.DiKTat')                             +'{{yes}}       | {{yes}}      | {{yes}}      | {{no}}  |',
```

Comment:
Please add diktat usage examples to `plugin-gradle.README` and `plugin-maven/README`